app/controllers/user/requests_controller.rb

```diff
@@ -13,6 +13,15 @@ def index
   # GET /requests/1
   def show
     authorize @request, :show?
+    @request.view
+  end
+
+  # POST /requests/1/decline
+  def decline
```

Comment:
could this be a `destroy` method. We can still call @request.decline, but it would map http methods in an expected way.